src/GitHub.App/ViewModels/PullRequestFileNode.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using GitHub.Models;
+
+namespace GitHub.ViewModels
+{
+    public class PullRequestFileNode : IPullRequestFileNode
+    {
+        public PullRequestFileNode(string fullPath, PullRequestFileStatus status)
+        {
+            FileName = System.IO.Path.GetFileName(fullPath);
+            Path = System.IO.Path.GetDirectoryName(fullPath);
+            Status = status;
+        }
+
+        public string FileName { get; }
+        public string Path { get; }
```

Comment:
Thing is that this is either a path to a file *or* a directory, as the property comes from the `IPullRequestChangeNode` which is implemented by `PullRequestDirectoryNode` as well as `PullRequestFileNode`. As such, I'm struggling to come up with a better name.